app/src/sharedTest/java/org/oppia/app/profile/AdminAuthActivityTest.kt

```diff
@@ -84,13 +87,35 @@ class AdminAuthActivityTest {
     }
   }
 
+  @Test
+  fun testAdminAuthActivity_inputCorrectPassword_opensAddAdministratorControlsActivity() {
+    launch<AdminAuthActivity>(
+      AdminAuthActivity.createAdminAuthActivityIntent(
+        context,
+        "12345",
+        0,
+        -10710042,
+        0
+      )
+    ).use {
+      onView(allOf(withId(R.id.input), isDescendantOfA(withId(R.id.input_pin)))).perform(
+        typeText("12345"),
+        closeSoftKeyboard()
+      )
+      onView(withId(R.id.submit_button)).perform(click())
+      intended(hasComponent(AdministratorControlsActivity::class.java.name))
+    }
+  }
+
   @Test
   fun testAdminAuthActivity_inputIncorrectPassword_checkError() {
     launch<AdminAuthActivity>(
       AdminAuthActivity.createAdminAuthActivityIntent(
         context,
         "12345",
-        -10710042
+        0,
+        -10710042,
+        0
```

Comment:
Pass enum value instead 
`AdminAuthEnum.PROFILE_ADD_PROFILE.value` so that code becomes more readable. Here and everywhere else.

Reply:
Done